### DISCUSSION TITLE
Indication for Favorites

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#7185] [Ideas]
Have you ever thought about having an indication that would indicates that a document has been added to Favorites?

**Comment by Frenzie** [comment]
Coincidentally somewhat related to https://www.mobileread.com/forums/showthread.php?p=4085334#post4085334 where I posted this general idea (favorites would be a star, of course):

![description-hint](https://user-images.githubusercontent.com/202757/105576172-3e217600-5d71-11eb-9dae-d5141541f6f5.png)

**Reply by ichnilatis-gr** [reply]
Yes, exactly.
Should I open a new issue thread?

**Reply by Frenzie** [reply]
We can probably wait for what @poire-z has to say about the considerations that went into it first. :-)

**Reply by poire-z** [reply]
Discussed previously at https://github.com/koreader/koreader/issues/6054#issuecomment-613880673. Please continue the discussion in #6054.
No real opinion. I'd probably have nothing against, as if one don't use favorites, we won't see anything more :) I just want this to add no overhead when there is no favorite.

**Comment by poire-z** [comment]
> where I posted this general idea 

About the "description hint", my idea (with this little bookmark on the side) came from what I've seen often in book shops: the book shop employees often write their opinion/review on a Post-It :tm: that they put either on the cover, or inside and making it overflow on the top or side, so people can see there is one, open the book, and read the employees' reviews/advices/opinion.
I thought hint for book description would be just fine with that appearance :) 

Your `(i)` is a bit too big for my liking - I usually don't care much about the descriptions, so it does not have to jump that much to my eyes :) but if I need to know, I can focus a bit and see that little bookmark.

**Reply by Frenzie** [reply]
> Your (i) is a bit too big for my liking

Of course it is! This is a very quickly slapped together example of the general principle, not remotely close to anything that looks good and proper. ;-)

**Reply by ichnilatis-gr** [reply]
So, should I repeat my request in [#6054](https://github.com/koreader/koreader/issues/6054)?

**Reply by poire-z** [reply]
Re-opened it. If you have any other thoughts on that, or visual suggestions, please put them in #6054.
